hudi-hadoop-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFormatWriter.java

```diff
@@ -275,12 +276,10 @@ private void addShutDownHook() {
     shutdownThread = new Thread() {
       public void run() {
         try {
-          LOG.warn("running logformatwriter hook");
-          if (output != null) {
```

Comment:
Why remove the check `output != null` ?

Reply:
> Why remove the check `output != null` ?

`closeStream` method already has the same logic: 
![Uploading image.png…]()